<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowLeader</class>
 <widget class="QDialog" name="MainWindowLeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frameWorkArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #6A5ACD, stop:1 #7B68EE);
    border-radius: 10px;
    border: 1px solid #6A5ACD;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <widget class="QComboBox" name="comboBoxlab2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #DDA0DD;</string>
    </property>
    <property name="placeholderText">
     <string>Выберите лабораторную</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxlab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #DDA0DD;</string>
    </property>
    <property name="placeholderText">
     <string>Выберите студента</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxtest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #DDA0DD;</string>
    </property>
    <property name="placeholderText">
     <string>Выберите студента</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxtest2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #DDA0DD;</string>
    </property>
    <property name="placeholderText">
     <string>Выберите тему</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxtest3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #DDA0DD;</string>
    </property>
    <property name="placeholderText">
     <string>Выберите оценку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowMatchingTemperament">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnLab">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>Отметить лабораторную работу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>Отметить проверочную работу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Выберите действие</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
